test.js

```diff
@@ -0,0 +1,276 @@
+      )
+    ).toBe(">block quote *bold* text\n\n");
+    expect(
+      slackify(
+        '<blockquote class="ghq-card-content__block-quote" data-ghq-card-content-type="BLOCK_QUOTE">block quote <em class="ghq-card-content__italic" data-ghq-card-content-type="ITALIC">italic</em> text</blockquote>'
+      )
+    ).toBe(">block quote _italic_ text\n\n");
+    expect(
+      slackify(
+        '<blockquote class="ghq-card-content__block-quote" data-ghq-card-content-type="BLOCK_QUOTE">block quote <u class="ghq-card-content__underline" style="text-decoration:underline" data-ghq-card-content-type="UNDERLINE">underline</u> text</blockquote>'
+      )
+    ).toBe(">block quote underline text\n\n");
+    expect(
+      slackify(
+        '<blockquote class="ghq-card-content__block-quote" data-ghq-card-content-type="BLOCK_QUOTE">block quote <del class="ghq-card-content__strikethrough" style="text-decoration:line-through" data-ghq-card-content-type="STRIKETHROUGH">strikethrough</del> text</blockquote>'
+      )
+    ).toBe(">block quote strikethrough text\n\n");
+    expect(
+      slackify(
+        '<blockquote class="ghq-card-content__block-quote" data-ghq-card-content-type="BLOCK_QUOTE">block quote <mark class="ghq-card-content__highlight" style="background-color:#fde892" data-ghq-card-content-type="HIGHLIGHT">highlight</mark> text</blockquote>'
+      )
+    ).toBe(">block quote highlight text\n\n");
+    expect(
+      slackify(
+        '<blockquote class="ghq-card-content__block-quote" data-ghq-card-content-type="BLOCK_QUOTE">block quote <span class="ghq-card-content__text-color" style="color:#9013fe" data-ghq-card-content-type="TEXT_COLOR">color</span> text</blockquote>'
+      )
+    ).toBe(">block quote color text\n\n");
+    expect(
+      slackify(
+        '<blockquote class="ghq-card-content__block-quote" data-ghq-card-content-type="BLOCK_QUOTE">block quote <a href="https://google.com" target="_blank" rel="noopener noreferrer" class="ghq-card-content__link" data-ghq-card-content-type="LINK">link</a></blockquote>'
+      )
+    ).toBe(">block quote <https://google.com|link>\n\n");
+    expect(
+      slackify(
+        '<blockquote class="ghq-card-content__block-quote" data-ghq-card-content-type="BLOCK_QUOTE">block quote <a target="_blank" rel="noopener noreferrer" class="ghq-card-content__file" data-ghq-card-content-type="FILE">file</a></blockquote>'
+      )
+    ).toBe(">block quote file\n\n");
+    expect(
+      slackify(
+        '<blockquote class="ghq-card-content__block-quote" data-ghq-card-content-type="BLOCK_QUOTE"><code class="ghq-card-content__code-snippet" data-ghq-card-content-type="CODE_SNIPPET">block quote guru code snippet</code></blockquote>'
+      )
+    ).toBe(">`block quote guru code snippet`\n\n");
+  });
+
+  it("full example", () => {
```

Comment:
I updated this text to actually use real guru html as input, the old test did not do this